RevitLookup/Application.cs

```diff
@@ -77,6 +78,7 @@ private void CreateRibbonPanel()
         var splitButton = ribbonPanel.AddSplitButton("RevitLookup", "Interaction");
 
         var splitDashboardButton = splitButton.AddPushButton<DashboardCommand>("Dashboard");
+        splitDashboardButton.SetAvailabilityController<AlwaysCommandAvailability>();
```

Comment:
The `SetAvailabilityController' is a pretty slow thing in the API.

Reply:
To be able to use RevitLookup with no document open.

Reply:
ah, okay, thanks for the clarification 😉